components/mdc/index.js

```diff
@@ -46,5 +47,6 @@ export {
   TopAppBar,
   Snackbar,
   Select,
-  setNotice
+  setNotice,
+  Menu
```

Comment:
Would you be open to putting this after "List", to keep this alphabetical?

(Since this is a release PR, feel free to address my comments in later feature branches. Sorry for not staying engaged enough to notice these when they were PR'd from feature branches.)

Reply:
yup, just PR'd it

Reply:
Just approved that PR

Reply:
merged